build/commands/scripts/sync.js

```diff
@@ -97,12 +97,11 @@ function buildDefaultGClientConfig() {
   if (process.env.GIT_CACHE_PATH) {
     out += toGClientConfigItem('cache_dir', process.env.GIT_CACHE_PATH)
   }
-  if (config.targetOS) {
-    out += toGClientConfigItem('target_os', [config.targetOS], false)
+  if (targetOSList) {
+    out += toGClientConfigItem('target_os', targetOSList, false)
   }
-  if (config.targetOS === 'linux') {
-    // Run hooks for Arm64. This in particular creates the arm64 sysroot.
-    out += toGClientConfigItem('target_cpu', ['arm64'], false)
+  if (targetArchList) {
+    out += toGClientConfigItem('target_cpu', targetArchList, false)
```

Comment:
`--target_arch=arm64` is passed on linux arm64 PR builders to `npm run sync` so this should work as expected.